src/sentry/issues/escalating_group_forecast.py

```diff
@@ -59,9 +59,6 @@ def fetch(cls, project_id: int, group_id: int) -> Optional[EscalatingGroupForeca
         results = nodestore.get(cls.build_storage_identifier(project_id, group_id))
         if results:
             return EscalatingGroupForecast.from_dict(results)
-        logger.exception(
-            f"Forecast does not exist for project id: {str(project_id)} group id: {str(group_id)}"
-        )
         generate_and_save_missing_forecasts.delay(group_id=group_id)
         return EscalatingGroupForecast(
```

Comment:
fetch will always return `ONE_EVENT_FORECAST`. 
based on initial investigation of the issue, a group will only not have a forecast in nodestore if has not been seen for 7 days or nodestore TTL has expired. thus, the event that triggered this function call will be the first event for this group, and thus the `ONE_EVENT_FORECAST` will be accurate. `generate_and_save_missing_forecasts` is also run in a task after